build_container/build_container_ubuntu.sh

```diff
@@ -44,9 +44,50 @@ apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
 
 apt-get update -y
 
-apt-get install -y --no-install-recommends docker-ce-cli wget make cmake git python python-pip python-setuptools python3 python3-pip \
-  python3-setuptools python3-yaml unzip bc libtool automake zip time gdb strace tshark tcpdump patch xz-utils rsync ssh-client \
-  google-cloud-sdk libncurses-dev doxygen graphviz python3.8 ninja-build bzip2 sudo
+PACKAGES=(
+    automake
+    bc
+    bzip2
+    cmake
+    docker-ce-cli
+    doxygen
+    gdb
+    git
+    google-cloud-sdk
+    graphviz
+    libncurses-dev
+    libtool
+    make
+    ninja-build
+    patch
+    python
+    python-pip
+    python-setuptools
+    python3
+    python3-pip
+    python3-setuptools
+    python3-yaml
+    python3.8
+    rsync
+    ssh-client
+    strace
+    sudo
+    tcpdump
+    time
+    tshark
+    unzip
+    wget
+    xz-utils
+    zip)
+
+# todo: remove this once kitware repo is fixed
```

Comment:
im wondering if a better approach is just to exclude the kitware repo on arm64, its currently "experimental" and i think we only add it for cmake

Reply:
I'm fine either way. What do you recommend?

Reply:
hopefully we can remove soon, so the way of less typing and as ive done it `8)`

Reply:
the downside to this way:

if the ubuntu  version changes it might break

Reply:
there are no downside to just  not including the repo *if we are not using it for anything else*

Reply:
OK let's try this and we can revisit.